File: beecrowd/javascript/1002.js
var input = require('fs').readFileSync('C:/Users/Boaz/Pictures/ARQUIVOS IMPORTANTES/javascript/URI/dev/stdin/file.txt', 'utf8');
var lines = input.split('\n');

var raio = Number(lines.shift());
var pi = 3.14159;
var area = (raio ** raio) * pi;

console.log(`A=${area}`);

File: beecrowd/C/1002.c
#include <stdio.h>
 
int main() {
    double area;
    double raio, pi = 3.14159;
    scanf("%lf", &raio);

    area = (double) (pi * (raio * raio));
    printf("A=%f\n", area);
 
    return 0;
}